atomsci/ddm/utils/struct_utils.py

```diff
@@ -455,7 +453,7 @@ def fix_moe_smiles(smiles):
     """
     protn_pat = re.compile(r'\[([cCBnNPS])([-\+])(@*)(H[1234]*)*\]')
     scalar = False
-    if type(smiles) == str:
+    if isinstance(smiles, str):
```

Comment:
test?

Reply:
I think this is a duplicate of the comment above.

Reply:
These tests should cover it. https://github.com/ATOMScience-org/AMPL/commit/8246be5a91e0e4999343198956c5085573903396